engine/execution/ingestion/engine.go

```diff
@@ -842,8 +843,8 @@ func newQueue(blockify queue.Blockify, queues *stdmap.QueuesBackdata) (*queue.Qu
 // G
 func enqueue(blockify queue.Blockify, queues *stdmap.QueuesBackdata) (*queue.Queue, bool) {
 	for _, queue := range queues.All() {
-		if queue.TryAdd(blockify) {
-			return queue, true
+		if stored, new := queue.TryAdd(blockify); stored {
+			return queue, new
```

Comment:
```suggestion
		if stored, isNew := queue.TryAdd(blockify); stored {
			return queue, isNew
```

`new` is a golang reserved keyword, better to use a different one